bridge/mumble/mumble.go

```diff
@@ -249,7 +249,7 @@ func (b *Bmumble) processMessage(msg *config.Message) {
 
 	// If there is a maximum message length, split and truncate the lines
 	var msgLines []string
-	if maxLength := b.serverConfig.MaximumMessageLength; maxLength != nil {
+	if maxLength := b.serverConfig.MaximumMessageLength; *maxLength != 0 {
```

Comment:
Why do you remove the nil check? If the pointer is nil, it'll panic

Reply:
I wasn't aware of that. This is my first time doing Golang.

Reply:
No problem, why did you close it?